examples failing to build in debug

> /home/peterh/bin/gcc-arm-none-eabi-10.3-2021.10/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: CMakeFiles/ht16k33_i2c.dir/ht16k33_i2c.c.obj: in function `main':
> 
> /home/peterh/source/pico/pico-examples3/i2c/ht16k33_i2c/ht16k33_i2c.c:166: undefined reference to `ht16k33_display_set'
> 
> /home/peterh/bin/gcc-arm-none-eabi-10.3-2021.10/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /home/peterh/source/pico/pico-examples3/i2c/ht16k33_i2c/ht16k33_i2c.c:167: undefined reference to `ht16k33_display_set'
> 
> /home/peterh/bin/gcc-arm-none-eabi-10.3-2021.10/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /home/peterh/source/pico/pico-examples3/i2c/ht16k33_i2c/ht16k33_i2c.c:168: undefined reference to `ht16k33_display_set'
> 
> /home/peterh/bin/gcc-arm-none-eabi-10.3-2021.10/bin/../lib/gcc/arm-none-eabi/10.3.1/../../../../arm-none-eabi/bin/ld: /home/peterh/source/pico/pico-examples3/i2c/ht16k33_i2c/ht16k33_i2c.c:169: undefined reference to `ht16k33_display_set'
> 
> collect2: error: ld returned 1 exit status
> 
> make[2]: *** [i2c/ht16k33_i2c/CMakeFiles/ht16k33_i2c.dir/build.make:823: i2c/ht16k33_i2c/ht16k33_i2c.elf] Error 1
> 
> make[1]: *** [CMakeFiles/Makefile2:5738: i2c/ht16k33_i2c/CMakeFiles/ht16k33_i2c.dir/all] Error 2
> 
> make: *** [Makefile:91: all] Error 2
> 
>

--- a/i2c/ht16k33_i2c/ht16k33_i2c.c
+++ b/i2c/ht16k33_i2c/ht16k33_i2c.c
@@ -94,7 +94,7 @@ void ht16k33_init() {
 }
 
 // Send a specific binary value to the specified digit
-inline void ht16k33_display_set(int position, uint16_t bin) {
+static inline void ht16k33_display_set(int position, uint16_t bin) {
     uint8_t buf[3];
     buf[0] = position * 2;
     buf[1] = bin & 0xff;
@@ -102,7 +102,7 @@ inline void ht16k33_display_set(int position, uint16_t bin) {
     i2c_write_blocking(i2c_default, I2C_addr, buf, count_of(buf), false);
 }
 
-inline void ht16k33_display_char(int position, char ch) {
+static inline void ht16k33_display_char(int position, char ch) {
     ht16k33_display_set(position, char_to_pattern(ch));
 }
     

